tests/test_env.py

```diff
@@ -43,6 +43,42 @@ def test_with_pip(self):
         self.assert_('foo' in e.dependencies['pip'])
         self.assert_('baz' in e.dependencies['pip'])
 
+    def test_with_selectors(self):
```

Comment:
Could you also check one of the platform selectors in this test? I think adding:

```
  - pywin32 # [win]
```

And adding an extra test like this:

```python
if sys.platform.startswith('win'):
    self.assert_('pywin32' in e.dependencies['conda'])
else:
    self.assert_('pywin32' not in e.dependencies['conda'])
```

would be enough